packages/node/src/indexer/api.wrapper.ts

```diff
@@ -174,4 +173,39 @@ export class ApiWrapper {
     }
     return lastHeight;
   }
+
+  async getBlockByHeight(height: number): Promise<SignedBlock | AlgorandBlock> {
+    let block: SignedBlock | AlgorandBlock;
+    switch (this.network) {
+      case 'algorand':
+        block = (await this.algorand.client.block(height).do()).block;
+        break;
+      case 'polkadot':
+        break;
```

Comment:
Il manque implementation Algorand là

Reply:
Edit: l'impl polkadot.
C"est juste utilisé par algorand, donc ça a rien à faire dans un wrapper qui switch.
Et c'est juste utilisé une fois dans la fontion juste en dessous. Donc ya pas besoin de le mettre dans un wrapper.
Tu prends juste `await this.algorand.client.block(height).do()).block` et tu le met dans fetchBlockArray